TGC/common/buildings.txt

```diff
@@ -732,15 +732,15 @@
 		# Audax Validator "." Ignore_NEXT
 		type = university
 		# Audax Validator "." Ignore_NEXT
-		cost = 30000
+		cost = 15000
```

Comment:
What is the purpose of this change? I require clarification

Reply:
change log